docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -80,6 +80,10 @@ using a ',' delimited list. The supported policies are:
     :ref:`config_http_filters_router_x-envoy-upstream-rq-timeout-ms` is an outer time limit for a
     request, including any retries that take place.
 
+gateway-error
+  Envoy will attempt a retry if the upstream server responds with 502, 503 or 504 response code. This policy should have
```

Comment:
Technically, all of the same documentation applies here as per the "5xx" policy. I would recommend rephrasing in the negative sense. E.g., "This policy is similar to the 5xx policy but will only retry requests that result in a 502, 503, or 504."